ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -34,30 +34,51 @@
 | -------- | -------- | -------- | -------- |
 {# ethernet interfaces #}
 {# ------- #}
-{%      if ethernet_interfaces is defined and ethernet_interfaces is not none %}
+{%      if ethernet_interfaces is arista.avd.defined %}
 {%          for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
-{%              if ethernet_interfaces[ethernet_interface].isis_enable is defined and ethernet_interfaces[ethernet_interface].isis_enable is not none %}
-| {{  ethernet_interface }} | {{ ethernet_interfaces[ethernet_interface].isis_enable }} |  {{ ethernet_interfaces[ethernet_interface].isis_metric | default('-') }} | {% if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is defined and ethernet_interfaces[ethernet_interface].isis_network_point_to_point is not none %} point-to-point {% elif ethernet_interfaces[ethernet_interface].isis_passive is defined and ethernet_interfaces[ethernet_interface].isis_passive is not none %} passive {% else %} - {% endif %}|
+{%              if ethernet_interfaces[ethernet_interface].isis_enable is arista.avd.defined %}
+| {{  ethernet_interface }} | {{ ethernet_interfaces[ethernet_interface].isis_enable }} |  {{ ethernet_interfaces[ethernet_interface].isis_metric | default('-') }} | {% if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is arista.avd.defined %} point-to-point {% elif ethernet_interfaces[ethernet_interface].isis_passive is arista.avd.defined %} passive {% else %} - {% endif %}|
 {%              endif %}
 {%          endfor %}
 {%      endif %}
 {# ------- #}
 {# vlan interfaces #}
 {# ------- #}
-{%      if vlan_interfaces is defined and vlan_interfaces is not none %}
+{%      if vlan_interfaces is arista.avd.defined %}
 {%          for vlan_interface in vlan_interfaces | arista.avd.natural_sort %}
-{%              if vlan_interfaces[vlan_interface].isis_enable is defined and vlan_interfaces[vlan_interface].isis_enable is not none %}
-| {{ vlan_interface }} | {{ vlan_interfaces[vlan_interface].isis_enable }} |  {{ vlan_interfaces[vlan_interface].isis_metric | default('-') }} | {% if vlan_interfaces[vlan_interface].isis_network_point_to_point is defined and vlan_interfaces[vlan_interface].isis_network_point_to_point is not none %} point-to-point {% elif vlan_interfaces[vlan_interface].isis_passive is defined and vlan_interfaces[vlan_interface].isis_passive is not none %} passive {% else %} - {% endif %}|
+{%              if vlan_interfaces[vlan_interface].isis_enable is arista.avd.defined %}
+| {{ vlan_interface }} | {{ vlan_interfaces[vlan_interface].isis_enable }} |  {{ vlan_interfaces[vlan_interface].isis_metric | default('-') }} | {% if vlan_interfaces[vlan_interface].isis_network_point_to_point is arista.avd.defined %} point-to-point {% elif vlan_interfaces[vlan_interface].isis_passive is arista.avd.defined %} passive {% else %} - {% endif %}|
```

Comment:
Should be arista.avd.defined instead of defined.

Interface mode: ```isis_network_point_to_point``` or ```isis_passive``` | arista.avd.defined tests should be true. Also please remove these big inline coding and replace them with simple variables

Reply:
Fixed.